src/pages/ReimbursementAccount/utils/mapCurrencyToCountry.ts

```diff
@@ -0,0 +1,18 @@
+import CONST from '@src/CONST';
+
+function mapCurrencyToCountry(currency: string): string {
+    switch (currency) {
+        case CONST.CURRENCY.USD:
+            return 'US';
```

Comment:
I think we already had constant values for these strings, let's use them instead:
ie: `CONST.COUNTRY.US`